lib/fennec/evaluator/channel_bind/request.ex

```diff
@@ -0,0 +1,147 @@
+defmodule Fennec.Evaluator.ChannelBind.Request do
+  @moduledoc false
+
+  alias Jerboa.Params
+  alias Jerboa.Format.Body.Attribute.ErrorCode
+  alias Jerboa.Format.Body.Attribute.XORPeerAddress, as: XPA
+  alias Jerboa.Format.Body.Attribute.ChannelNumber
+  alias Fennec.UDP
+  alias Fennec.TURN
+  alias Fennec.TURN.Channel
+
+  import Fennec.Evaluator.Helper, only: [maybe: 3, maybe: 2]
+
+  require Logger
+
+  @lifetime 10 * 60 # 10 minutes
```

Comment:
We can't quite use `:erlang.minutes/1` coz it gives us milliseconds but we could write a `Fennec.Time.minutes/1` of our own.